docs/release-notes/8.1.0-1.md

```diff
@@ -0,0 +1,33 @@
+# Percona XtraBackup 8.1.0-1 (2023-10-26)
+
+Get started with [Quickstart Guide for Percona XtraBackup](..//quickstart-overview.md).
+
+[Percona XtraBackup (PXB)](https://www.percona.com/software/mysql-database/percona-xtrabackup) is a 100% open source backup solution for all versions of Percona Server for MySQL and MySQL® that performs online non-blocking, tightly compressed, highly secure full backups on transactional systems. Maintain fully available applications during planned maintenance windows with Percona XtraBackup.
+
+## Release highlights
+
+Percona XtraBackup 8.1.0-1 is based on MySQL 8.1 and fully supports the Percona Server for MySQL 8.1 Innovation series and the MySQL 8.1 Innovation series. This release allows taking backups of Percona Server 8.1.0-1 and MySQL 8.1.
+
+Use the Percona XtraBackup 8.0 series to take backups of Percona Server for MySQL 8.0.x or MySQL 8.0.x. Percona XtraBackup 8.1.0-1 does not take a backup of the Percona Server for MySQL 8.0 or MySQL 8.0.x series.
+
+## Improvement
+
+* [PXB-3155] : Moved the vault from a plugin to a component.
+
+## Deprecated or removed
+
+The `--stats` mode of operation for the xtrabackup binary has been removed.
+
+For keyring variations except the `keyring_file`, Percona XtraBackup 8.1.0-1 supports only the component versions. The `keyring_file` plugin and component is supported. The `keyring_file` plugin is deprecated in MySQL and Percona Server for MySQL and may be removed in the future.
```

Comment:
Looks good. One minor suggestion. Instead of saying variants, maybe we can list the names explicitly. We only have three of them.

For keyrings KMIP and KMS, only component versions are supported. For keyring_file, both plugin and component versions are supported. The keyring_file plugin is deprecated...